src/runtime/testing/slot-test-observer.ts

```diff
@@ -150,7 +150,9 @@ export class SlotTestObserver extends AbstractSlotObserver {
     if (this.onlyOptionalExpectations()) {
       return Promise.resolve();
     }
-    return new Promise((resolve, reject) => this.onExpectationsComplete = resolve);
+    return new Promise((resolve, reject) => {
+      this.onExpectationsComplete = resolve;
+    });
```

Comment:
This seems like a case of the linter being overly restrictive, I'm not blocking on it, but maybe we should relax the lint rule?

Reply:
I'm not sure that is possible.